brave/brave-core
# Move P3aOnoardingActivity to extend FirstRunActivityBase

Resolves brave/brave-browser/issues/16244.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Install the new version apk on a new device
2. Check the onboarding page looks the same as previous version
